Topics/User_Experience.md

```diff
@@ -21,6 +21,41 @@ ___Established Norms___
 ___Color Theory___
 - [Colors Influence Choice](https://usabilitygeek.com/colour-user-experience-psychology/#:~:text=Colour%20plays%20a%20crucial%20role,and%20identified%20with%20your%20industry.): Users are shown certain colors to enforce actions. For example, red is commonly linked to aggressive or bad emotions and as such can be used as a cancel to dissuade people from refunding items.
 
+### A Brief Summary of Common UX Principles
+
+#### Accessibility
+Accessibility is a concept that encapsulates one central idea: to allow all users to easily enjoy and use applications. In common usage, it can refer to users with temporary or permanent disabilities (visual, motor, auditory, etc.), but is broader than this, referring also to overall robustness and operability (i.e. is the workflow too complicated? Can multiple devices access the application, if it is a web application?). Consideration for varying degrees of ability is fundamental to design—a product should reach as wide of an audience as possible. Oftentimes, accesibility considerations lead to unintended benefits. For example, providing closed captioning on web video content can improve SEO indexing.
```

Comment:
```suggestion
Accessibility is a concept that encapsulates one central idea: to allow all users to easily enjoy and use applications. In common usage, it can refer to users with temporary or permanent disabilities (visual, motor, auditory, etc.), but is broader than this, referring also to overall robustness and operability (i.e. is the workflow too complicated? Can multiple devices access the application, if it is a web application?). Consideration for varying degrees of ability is fundamental to design—a product should reach as wide of an audience as possible. Oftentimes, accessibility considerations lead to unintended benefits. For example, providing closed captioning on web video content can improve SEO indexing.
```